### ISSUE TITLE
is there a corresponding article for this code?

### ISSUE THREAD

**Issue by ddr-152** [XinyaoQiu/DRL-for-edge-computing#5]
Hello, author, is there a corresponding article for this code? There are some details that I can't understand. I want to learn more about the relevant details in conjunction with the paper. If you can provide a link to the paper, I would be very grateful.

**Comment by XinyaoQiu** [comment]
Yes, our article `DRL-for-edge-computing.pdf` is in the repository.

**Comment by ddr-152** [comment]
Thank you very much!




孙定蜜
***@***.***







                       
Original Email
                       
                     

Sender:"XinyaoQiu"< ***@***.*** &gt;;

Sent Time:2023/8/29 21:31

To:"XinyaoQiu/DRL-for-edge-computing"< ***@***.*** &gt;;

Cc recipient:"ddr-152"< ***@***.*** &gt;;"Author"< ***@***.*** &gt;;

Subject:Re: [XinyaoQiu/DRL-for-edge-computing] is there a correspondingarticle for this code? (Issue #5)




 
Closed #5 as completed.

**Comment by ddr-152** [comment]
![f70e75a09a6ab6dad1837b417a9bc4e](https://github.com/XinyaoQiu/DRL-for-edge-computing/assets/67107581/f2f4d51d-eb1f-4ee3-a659-768f4ad80b5c)
hi，Do the train.py files and evaluate.py files mentioned here need to be generated according to the command? I didn't see these two files in the code base, looking forward to your answer!

**Comment by ddr-152** [comment]
> Yes, our article `DRL-for-edge-computing.pdf` is in the repository.

Thank you very much!

**Comment by XinyaoQiu** [comment]
> ![f70e75a09a6ab6dad1837b417a9bc4e](https://user-images.githubusercontent.com/67107581/264246364-f2f4d51d-eb1f-4ee3-a659-768f4ad80b5c.png) hi，Do the train.py files and evaluate.py files mentioned here need to be generated according to the command? I didn't see these two files in the code base, looking forward to your answer!

Oh that's an out-of-date version. Maybe you can try to run `python3 run_ddpg.py` for example, but actually I'm not sure whether it works.

**Comment by ddr-152** [comment]
Hello, I have a lot of doubts about the problems in the code. I would like to ask if it is convenient to add you a contact information. My contact information is as follows: 3360730446@qq.com. I look forward to your reply.

**Comment by Yzhtc** [comment]
Hello, what is the value of the __init__ module noise and bound parameter in the python file of MAA2C? looking forward to your answer!
****
    def __init__(self, env, n_agents, state_dim, action_dim, action_lower_bound, action_higher_bound,
                 noise, bound, memory_capacity=10, max_steps=None,
                 roll_out_n_steps=10, tau=300, 
                 reward_gamma=0.99, reward_scale=1., done_penalty=-10,
                 actor_output_act=nn.functional.softmax, critic_loss="huber",
                 actor_lr=0.01, critic_lr=0.01, training_strategy="centralized",
                 optimizer_type="rmsprop", entropy_reg=0.00,
                 max_grad_norm=None, batch_size=10, episodes_before_train=0,
                 use_cuda=False, actor_parameter_sharing=False, critic_parameter_sharing=False, 
                 epsilon_start=0.9, epsilon_end=0.05, epsilon_decay=100):
